pkg/components/components.go

```diff
@@ -516,6 +516,7 @@ func GetCSVBase(name, displayName, description, image, replaces string, version
 				"alm-examples":   string(almExamples),
 				"capabilities":   "Full Lifecycle",
 				"certified":      "false",
+				"olm.targetNamespaces": "kubevirt-hyperconverged",
```

Comment:
Please rebase this and use hcCRNamespace variable as on line 507 so that we can render it differently from upstream to downstream